keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -603,6 +586,8 @@ Files:
 - test/integration/servicecidr/allocator_test.go
 - test/integration/servicecidr/migration_test.go
 - test/integration/servicecidr/servicecidr_test.go
+- test/integration/servicecidr/feature_enable_disable_test.go
+- test/integration/servicecidr/perf_test.go
 
 ##### e2e tests
```

Comment:
Are there more e2e-s planned? From checking the codebase I see only one currently. For starters I'm missing e2e covering GA API endpoints (see my earlier comment).

Reply:
Based on the [GA criteria ](https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/1880-multiple-service-cidrs/README.md#ga) I'm missing all 3 elements pointed out in this update:

> * 2 examples of real-world usage
> * More rigorous forms of testing—e.g., downgrade tests and scalability tests
> * Allowing time for feedback

Reply:
> 2 examples of real-world usage

I didn't know how to do this, since this is an opt-in feature is not possible to get telemetry, I know there are GKE customers using it and also that Kops is able to do it, see the description of this PR to find the examples I found of people using or testing it.

> More rigorous forms of testing—e.g., downgrade tests and scalability tests

This is a core feature, means that once enable it inherit all the scalability testing, upgrade/downgrade is added as integration

> Allowing time for feedback

It went beta in 1.31 and we usually leave one release for feedback, I got good internal feedback from GKE users .... I also want to avoid permanent betas

Reply:
should I update this information on the doc or is enough with the description on the issue? I linked several places that I think should be proof the feature is being used in production ... we generally add the diversity of implementations to avoid favoring vendors, but in this case this feature is a core functionality that will be used everywhere

Reply:
For posterity, discussed this with Antonio on slack, I suggested to link the public references (Kops from what he writes above, plus mention GKE customers).